USER_MANUAL.md

```diff
@@ -1549,58 +1740,983 @@ The following figure offers a visual guideline for the various parameters that c
+
+#### repair_delay (Source file)
+
+**Description:** This is a propagating parameter that can be set at multiple level of granularity. See [repair delay](#values-delaypropagating-parameter) for more details on what this parameter represents. In this file, this is a single numeric value in days.
+
+This parameter can also be set in the following files:
+
+- [Virtual World Parameter file](#values-delaypropagating-parameter)
+- [Site File](#repairable_repair_delay-sites-file)
+- [Site type File](#repairable_repair_delay-site-type-file)
+- [Equipment File](#repairable_repair_delay-equipment-file)
+
+#### repair_cost (Source file)
+
+**Description:** This is a propagating parameter that can be set at multiple level of granularity. See [repair cost](#values-cost-propagating-parameter) for more details on what this parameter represents. In this file, this is a single numeric value, representing cost.
+
+This parameter can also be set in the following files:
+
+- [Virtual World Parameter file](#values-cost-propagating-parameter)
+- [Sites File](#repairable_repair_cost-sites-file)
+- [Site type File](#repairable_repair_cost-site-type-file)
+- [Equipment File](#repairable_repair_cost-equipment-file)
+
+#### emissions_rate_source (Source file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See [emissions_rate_source](#emissions_rate_source-propagating-parameter) for more details on what this parameter represents. In this file this is a user defined value, that corresponds to the [source](#header) of the emissions for the given emission component-source. This case-sensitive value specifies which of the potential emissions characteristics in the [emissions file](#emissions-file) should be used for the given emissions rate source.
+
+This parameter can also be set in the following files:
+
+Repairable Emissions:
+
+- [Virtual World Parameter file](#emissions_rate_source-propagating-parameter)
+- [Sites File](#repairable_emissions_rate_source-sites-file)
+- [Site type File](#repairable_emissions_rate_source-site-type-file)
+- [Equipment File](#repairable_emissions_rate_source-equipment-file)
+
+Non-Repairable Emissions:
+
+- [Virtual World Parameter file](#emissions_rate_source-propagating-parameter)
+- [Sites File](#non_repairable_emissions_rate_source-sites-file)
+- [Site type File](#non_repairable_emissions_rate_source-site-type-file)
+- [Equipment File](#non_repairable_emissions_rate_source-equipment-file)
+
+#### emissions_production_rate (Source file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See [emissions_production_rate](#emissions_production_rate-propagating-parameter) for more details on what this parameter represents. In this file, this is a single numeric value, representing the emission production rate of the given source at the given component per day.
```

Comment:
This one didn't get updated to be in line with the new emissions rate source descriptions